lesson1/exercise1.md

```diff
@@ -14,9 +14,9 @@ Write them down/remember them to share later with the class.
 {:toc}
 
 ## Precondition
-You should have opened the [repl.it](https://repl.it/) website and created a new *Java* repl (do not yet change things).
+You should have opened the [codeboard.io](https://codeboard.io/) website and created a new *Java* codeboard (do not yet change things).
 
-If not, please visit [this page](replit) to find out how and come back here after that.
+If not, please visit [this page](https://codeboard.io/) to find out how and come back here after that.
```

Comment:
I don't understand the point of this sentence. It just links to the same page as the sentence above, no?

Reply:
Yes, removed